packages/server-side-graphql-client/lib/server-side-graphql-client.js

```diff
@@ -2,7 +2,11 @@ const runQuery = async ({ keystone, query, variables, context }) => {
   const { data, errors } = await keystone.executeGraphQL({
     context:
       context ||
-      keystone.createContext({ schemaName: 'public', authentication: {}, skipAccessControl: true }),
+      keystone.createContext({
```

Comment:
This file shouldn't have changed.

Reply:
I think it happens due to `yarn format`. 🤔